File: benchmark/for/for.cpp
#include <iostream>
#include <time.h>

int main(){

  long sum = 0;
  int N = 1000000;
  int i;
  clock_t start = clock();
  for(i = 0; i < N; i++){
    sum += i;
  }
  clock_t end = clock();

  std::cout << "C++ " << sum << " " << (double)(end-start)/CLOCKS_PER_SEC << "sec." << std::endl;
}

File: benchmark/for/For.java
public class For{
  public static void main(String args[]){
    int N = 1000000;
    long sum = 0;
    long start = System.nanoTime();
    for(int i = 0; i < N; i++){
      sum += i;
    }
    long time = System.nanoTime() - start;
    System.out.println("java "+sum+" "+time/1000000000.+"sec.");
  }
}
